.github/CHANGELOG.md

```diff
@@ -21,6 +21,9 @@
 
 ### Bug fixes
 
+* Update the CMake internal references to enable sub-project compilation with affecting the parent package.
```

Comment:
I know why you did it, but I'm not getting what you meant by "with affecting the parent package.".
Could you maybe rephrase that?

Reply:
That was a hastily written message indeed. Let's push ahead for the GPU fixes, and I'll make it better for release